CMakeLists.txt

```diff
@@ -103,6 +103,8 @@ option(PCAPPP_PACKAGE "Package Pcap++ could require a recent version of CMake" O
 set(CMAKE_CXX_STANDARD 11)
 # popen()/pclose() are not C++ standards
 set(CMAKE_CXX_EXTENSIONS ON)
+# Set Position Independent Code for static libraries
+set(CMAKE_POSITION_INDEPENDENT_CODE ON)
```

Comment:
Maybe we should have a default value (`ON` or `OFF`?), but allow the user to control it?

Reply:
I don't see a case where you want to disable it, and it was built with fPIC for a very long time and nobody complains.

It's easy to add an option but not sure it worth it.

Reply:
ok 👍